src/client/src/shared/helpers/password-toggle.ts

```diff
@@ -0,0 +1,29 @@
+import { dom } from "../libs/modules/dom/dom";
```

Comment:
```suggestion
import { dom } from "../libs/modules/dom/dom.js";
```

include js extension in all of the imports